docs/pages/resources/fundamentals-libraries.md

```diff
@@ -0,0 +1,22 @@
+---
+title: SAP Fiori Fundamentals Libraries
+keywords: libraries, frameworks
+sidebar: left-navigation-sidebar
+toc: false
+permalink: /fundamentals-libraries/
```

Comment:
This routing isn't consistent - see the other files inside the resources folder for an example.